WeakAuras/Prototypes.lua

```diff
@@ -876,6 +876,16 @@ WeakAuras.load_prototype = {
        end,
       test = "WeakAuras.CheckNumericIds([[%s]], zoneId)"
     },
+    {
```

Comment:
So which of those is the more useful one? The Zone Group or the Zone Id?

That one should be first.

Reply:
zoneId can always return something so it's probably more useful? (an ID that is)